cypress/support/shared.ts

```diff
@@ -0,0 +1,14 @@
+import { SHARED } from "cypress/elements/shared";
+
+export function waitForProgressBarToNotBeVisible() {
+  cy.get(SHARED.spinner).should("not.exist");
+}
+
+export function filterProducts(firstElement, secondElement) {
```

Comment:
Variables firstElement, secondElement should be better named. Maybe filterProductsBy, and selectedFilter

Reply:
yup, we should avoid using names which doesn't describe what the first element is